holoviews/ipython/widgets.py

```diff
@@ -506,20 +340,56 @@ def __call__(self):
 
 
 
-class SelectionWidget(ScrubberWidget):
+class CustomCommSocket(CommSocket):
+    """
+    A CustomCommSocket is required to suspend displaying the nbagg
+    output until the HTML has been rendered by the widget.
+    """
```

Comment:
I still have no idea what this does. Either needs more documentation or some link to what this is about.